development/mimir-microservices-mode/docker-compose.jsonnet

```diff
@@ -114,7 +114,7 @@ std.manifestYamlDoc({
         extraArguments:
           // Use of scheduler is activated by `-querier.scheduler-address` option and setting -querier.frontend-address option to nothing.
           if $._config.use_query_scheduler then '-querier.scheduler-address=query-scheduler:9011 -querier.frontend-address=' else '',
-        extraVolumes: ['./config/mimir_override.yaml:/mimir/config/mimir.yaml'],
+        extraVolumes: ['./config/mimir.yaml:/mimir/config/mimir.yaml'],
```

Comment:
These can be removed, I think. By default it has the correct volumes ([Ref](https://github.com/grafana/mimir/blob/b8772e6a759d886dc454ae14c9b944c575254567/development/mimir-microservices-mode/docker-compose.jsonnet#L272)) , as part of the mimirService construtor

Reply:
Will do.